### DISCUSSION TITLE
how to inspect highlight in Telescope window

### DISCUSSION THREAD

**Discussion by Arpple** [LazyVim/LazyVim#2840] [Q&A] [answered]
normally in text buffer, I can use :Inspect to see what the current highlight the text is using. 

But for temporary window like Telescope I don't know how I can select the specific text and run :Inspect on it. 

Is there any other way to know what highlight that part is using? For example in my case, I want to know what highlight the search hit character is using (blue in my case)
![image](https://github.com/LazyVim/LazyVim/assets/12965313/ebe8c64a-36b8-4eca-b0ae-af8024cac8c1)

**Comment by dpetka2001** [comment, marked as answer]
If you do `<leader>sH` to search for highlight groups and then write `Telescope` it will show you all the Telescope highlight groups. From there you try and deduce what the correct highlight group might be. I tried with `TelescopeMatching` and it seems to do what you want. I'm not aware how you can get the cursor inside the results window so that you can inspect.
